app/database.js

```diff
@@ -67,6 +73,16 @@ define(['sqlite', 'app/config', 'jquery'], function (sqlite, config) {
     }
     return true;
   }
+	
+	// Auxiluary uniqueness function
+  function isUnique2(unique_name1, unique_name2, queryResult, row) {
```

Comment:
Explain the difference between isUnique and isUnique2 by choosing different function names. Or overload the function, well js doesn't support overloading, but [this]http://stackoverflow.com/a/457589/1357229) seems like a good solution.